packages/dev-scripts/examples/template-react/package.json.template.tsx

```diff
@@ -8,9 +8,8 @@ const template = (project: Project) => ({
   scripts: {
     start: "vite",
     dev: "vite",
-    build: "tsc && vite build",
-    preview: "vite preview",
-    lint: "eslint . --max-warnings 0",
+    "build:prod": "tsc && vite build",
```

Comment:
you named this build:prod because you don't want it included when running build in root, right?

Reply:
Exactly right, pnpm run build should probably not build each example, just the packages